ui/src/index.html

```diff
@@ -5,6 +5,8 @@
 	<meta charset="utf-8">
 	<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
 	<link rel="shortcut icon" type="image/svg+xml" href="/static/assets/favicon.svg" />
+	<link rel="apple-touch-icon" href="/static/assets/apple-touch-icon.png" />
```

Comment:
Where is this icon, its not in your commit.

Reply:
added.